src/OpenTelemetry.Instrumentation.Owin/Implementation/OwinInstrumentationMetrics.cs

```diff
@@ -0,0 +1,31 @@
+// <copyright file="OwinInstrumentationMetrics.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics.Metrics;
+using System.Reflection;
+
+namespace OpenTelemetry.Instrumentation.Owin.Implementation;
+
+internal static class OwinInstrumentationMetrics
+{
+    internal static readonly AssemblyName AssemblyName = typeof(OwinInstrumentationMetrics).Assembly.GetName();
+
+    public static string MeterName => AssemblyName.Name;
+
+    public static Meter Instance => new Meter(MeterName, AssemblyName.Version.ToString());
+
+    public static Histogram<double> HttpClientDuration => Instance.CreateHistogram<double>("http.server.duration", "ms", "Measures the duration of inbound HTTP requests.");
```

Comment:
Use updated convention here? https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration

Reply:
@vishweshbankwar done!

Reply:
Should we update tracing too?

Reply:
No, I don't think tracing should be updated. From [this note](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/http):

> Existing HTTP instrumentations that are using [v1.20.0 of this document](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/trace/semantic_conventions/http.md) (or prior):
> 
> * SHOULD NOT change the version of the HTTP or networking conventions that they emit until the HTTP semantic conventions are marked stable [...]

The goal is to avoid breaking existing HTTP instrumentations until HTTP semantic conventions are stable (to ensure instrumentations only break once).